Packs/Zimperium/Classifiers/classifier-Zimperium.json

```diff
@@ -0,0 +1,16 @@
+{
+	"id": "Zimperium",
+	"name": "Zimperium - Classifier",
+	"type": "classification",
+	"description": "Classifies Zimperium incidents.",
+	"defaultIncidentType": "",
```

Comment:
no default incident type?

Reply:
Not AFAIK, they can choose to use the Zimperium event incidenttype, but not sure we want it as default. @Arsenikr WDYT?